pages/rdm_cycle/sharing.md

```diff
@@ -23,35 +23,33 @@ Even though it may not be possible to openly share all data because of ethical,
+  - If there are Discipline-specific repositories available for you data, this should be your primary choice. They will work towards a high level of FAIRness by recommending appropriate community standards for describing the data. 
+  - If there are no suitable Discipline-specific repositories for your data:
+    - Deposit the data in an _Institutional repository_, if there is one. These often provide stewardship and curation, helping to ensure that your dataset is preserved and accessible. Contact the Research Data Office function at your institution, if there is one.
     - Deposit the data in a [_General purpose repository_](https://www.nature.com/sdata/policies/repositories#general).
-* Consider depositing your data to a repository as early as possible. This saves a lot of trouble later on. Data can be put under embargo until you want to release it, e.g. at the time of article publication.
-* To make your life easier, structure the metadata according to your target repositories early on in the project (see Planning).
+* Structure the metadata according to your target repositories early on in the project (see [Planning](planning)).
 * Note that all data might not be possible to make publicly available because of ethical, legal, contractual, or intellectual property reasons. 
   - For some types of data, there are repositories that can harbour data that will be under controlled access.
-    - For the sharing of human-subject phenotype/genotype data there are well-established repositories ([EGA](https://ega-archive.org/), [dbGAP](https://www.ncbi.nlm.nih.gov/gap/), [JGA](https://www.ddbj.nig.ac.jp/jga/index-e.html)). 
-      - The choice will depend on the conditions under which data has been donated and the legislative/geographic boundaries for the research project/consortium. 
-    - There are also community-specific data hubs that may implement a controlled-access model e.g. ORPHANET, MJFF.
+   - The choice will depend on the conditions under which data has been donated and the legislative/geographic boundaries for the research project/consortium. 
+   - There are also community-specific data hubs that may implement a controlled-access model.
   - If there isn't any suitable repository that can harbour your controlled access data, it is recommended that you at least create a metadata record for the data in an Institutional or General purpose repository.
-  
+* Collaborative project partners need to agree on common data organisation, data formats and standards
 
-## Where can training materials and events about Data Sharing be found?
 
-{% include tess.html search="Data Sharing" %}
-
-## Related topics
+## Problems to be addressed in this stage
 
 {% include pagelist.html tag="share" %}
 
+
+## Available training material
```

Comment:
The title of the paragraph about training is still: Where can training materials and events about Data Sharing be found? or WP2 wants to change that?